WordPress/Classes/ViewRelated/Notifications/Views/LongPressGestureLabel.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+/// Subclass of UILabel that includes a long press gesture recognizer.
+/// The action attached to the recognizer will be set by the creator of the label.
+
+class LongPressGestureLabel: UILabel {
+    var longPressAction: (() -> Void)? {
+        didSet {
```

Comment:
I know this is an edge-case, but it seems like setting a different `longPressAction` on the same label would cause us to add the same recognizer multiple times, which could bite us in the future in cases like cell-reuse.

Reply:
So since the gesture recognizer itself is lazy it will only be initialized once. I tested this by adding another long press action and this is the array of gesture recognizers on the label:
 Optional<Array<UIGestureRecognizer>>
  ▿ some : 2 elements
    - 0 : <UITapGestureRecognizer: 0x60000220d000; state = Possible; view = <WordPress.LongPressGestureLabel 0x7f83f78bab10>; target= <(action=detailsWasPressed, target=<WordPress.NoteBlockCommentTableViewCell 0x7f83f685ba00>)>>
    - 1 : <UILongPressGestureRecognizer: 0x7f83f7f06140; state = Possible; view = <WordPress.LongPressGestureLabel 0x7f83f78bab10>; target= <(action=longPressAction:, target=<WordPress.LongPressGestureLabel 0x7f83f78bab10>)>>

 The UILongPressGestureRecognizer is only added once even though I called it twice.